database/migrations/2021_01_28_100545_create_students_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
this file was generated using the command

`php artisan make:migration create_students_table`